website/project/model.py

```diff
@@ -2403,7 +2403,7 @@ def add_unregistered_contributor(self, fullname, email, auth,
         self.save()
         return contributor
 
-    def set_privacy(self, permissions, auth=None):
+    def set_privacy(self, permissions, auth=None, logging=True):
```

Comment:
For consistency with other methods, this should be `log`, not `logging`. This will also prevent shadowing the `logging` import.